src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/PropertyNameProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+    /// <summary>
+    /// Contains constants representing the property names used in the ApplicationPropertyPage folder.
+    /// </summary>
+    internal class PropertyNameProvider
```

Comment:
I suggest calling this `PropertyNames` instead of `PropertyNameProvider`. A "provider" in CPS and dotnet/project-system tends to mean a factory that produces instances of some other type; e.g. an `IProjectPropertiesProvider` produces instances of `IProjectProperties`.

Reply:
Sounds good, adding the suggestion on my next commit.